README.md

```diff
@@ -1,12 +1,59 @@
-# CarCare
-Junior Design project repo for Team LTT
+# Car Care
+## Introduction
+Car Care collects diagnostic information from a car's OBD port and displays it on the user's smartphone with various graphics. The data can be viewed in real time or at a later date without requiring an active connection to the OBDII adapter. Alerts can be set up to notify the user when data received is lower or higher than a specified value.
 
-This repository is for the Android app version of Car Care.
-It is developed with Android Studio.
+Car Care is compatible with the ELM 327 OBDII adapter.
 
-Car Care collects diagnostic information from a car's OBD port and displays it on the user's smartphone with various graphics.
+## Release notes for version 1.0.0
+### New software features for this release
+* Added Google Services integrated login
+* Added Bluetooth pairing ability between the application and an OBDII bluetooth device
+* Added settings page to customize data shown on Home page
+* Added Vehicle Information page for storing vehicle information
+* Added static data card on Home Activity that displays current live values received from the OBD adapter
+* Added dynamic data card on Home Activity that displays a live history of values received from the OBD adapter
+* Added Dynamic Statistics page to view a history of values from previous trips
+* Added a trip filter to the Dynamic Statistics page to view data from specific previous trips
+* Added date reminders that trigger an alert when the current date is after the date specified in the reminder
+* Added value reminders that trigger an alert when the application receives a value for a specified type of data that sastisfies the  comparison specified in the reminder
+* Added the ability to archive reminders, allowing for deactivation and history of reminders
 
-It is compatible with the ELM 327 OBDII adapter.
+### Bug fixes made since last release
+* Fixed app crash when vehicles database was queried but no vehicles were present
 
```

Comment:
Add another bug fix for the app crashing if the Bluetooth adapter wasn't available.